fem/tmop.hpp

```diff
@@ -664,6 +664,9 @@ class TargetConstructor
    /// Used by target type IDEAL_SHAPE_EQUAL_SIZE. The default volume scale is 1.
    void SetVolumeScale(double vol_scale) { volume_scale = vol_scale; }
 
+   /// Checks if the target matrices contain non-trivial size specification.
+   bool ContainsVolumeInfo() const;
```

Comment:
This method should probably be virtual to support the overriding mentioned in the class's documentation header.  Speaking of which, that header should be updated to mention this method as well.

Reply:
Fixed.